src/main/java/seedu/address/model/util/DateTime.java

```diff
@@ -80,8 +82,13 @@ public static boolean isValidDateTimeRange(String test) {
         try {
             DateFormat df = new SimpleDateFormat("dd-MM-yyyy HH:mm");
             df.setLenient(false);
-            df.parse(test);
-            return true;
+            Date date = df.parse(test);
+            Date earliestDate = df.parse(EARLIEST_DATE_STRING);
```

Comment:
can it be EARLIEST_DATE instead of EARLIEST_DATE_STRING? so don't need to parse each time
also why Date and not LocalDateTime?